editor/editor_inspector.cpp

```diff
@@ -2429,6 +2429,13 @@ void EditorInspector::update_tree() {
 		_parse_added_editors(main_vbox, ped);
 	}
 
+	bool is_instance_node = false;
+
+	Node *node = Object::cast_to<Node>(object);
+	if (node) {
+		is_instance_node = node->get_scene_file_path() != "" && node->get_owner() != nullptr;
```

Comment:
```suggestion
		is_instance_node = !node->get_scene_file_path().is_empty() && node->get_owner() != nullptr;
```
This is preferred style for checking empty strings.